frontend/src/helpers/time-helper.js

```diff
@@ -0,0 +1,10 @@
+const formatAsHoursMinutes = minutes => {
+  const num = minutes;
+  const hours = num / 60;
+  const floorHours = Math.floor(hours);
+  const min = (hours - floorHours) * 60;
+  const roundMinutes = Math.round(min);
+  return floorHours + ' hour(s) and ' + roundMinutes + ' minute(s)';
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).